spec/migrations/20151026204114_add_vmware_ro_datastores_to_hosts_storages_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "spec_helper"
+require_migration
+
+describe AddVmwareRoDatastoresToHostsStorages do
+  let(:hosts_storages_stub) { migration_stub(:HostsStorage) }
+
+  migration_context :up do
+    it "Adds ID in correct region" do
+      migrate
```

Comment:
could you put an expectation into this test?
Maybe something like:

```ruby
it "Adds ID" do
  hosts_storages_stub.create
  migrate
  expect(hosts_storages_stub.reloat.id).to be
```

You may need to reload the caches to ensure the id was added to the table.